oidc/config.go

```diff
@@ -269,6 +287,9 @@ func (c *Config) Validate() error {
 			return fmt.Errorf("%s: %w", op, ErrInvalidCACert)
 		}
 	}
+	if c.ProviderCA != "" && c.RoundTripper != nil {
+		return fmt.Errorf("%s: you cannot specify both a ProviderCA and Transport: %w", op, ErrInvalidParameter)
```

Comment:
nit: Might replace `Transport` with the config field name `RoundTripper`? Could imagine looking for a config `Transport` field upon receiving this error message and being confused.

Reply:
ty!